pkgs/by-name/fr/framework-system-tools/package.nix

```diff
@@ -0,0 +1,51 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, rustPlatform
+, cargo
+, rustc
+, gnumake
+, pkg-config
+, udev
+}:
+
+let
+  rev = "491a587342a5d79366a25d803b7065169314279c";
+in rustPlatform.buildRustPackage {
+  pname = "framework-system-tools";
+  version = "unstable-2023-11-14";
+
+  src = fetchFromGitHub {
+    owner = "FrameworkComputer";
+    repo = "framework-system";
+    inherit rev;
+    hash = "sha256-qDtW4DvY19enCfkOBRaako9ngAkmSreoNWlL4QE2FAk=";
+  };
+
+  nativeBuildInputs = [
+    cargo
+    rustc
+    gnumake
+    pkg-config
+  ];
+
+  buildInputs = [
+    udev
+  ];
+
+  cargoLock = {
+    lockFile = ./Cargo.lock;
```

Comment:
Would always be better if we can convince upstream to switch to crates.io releases. Maybe ask them nicely, but it should not block the merging of this PR.

Reply:
Upstream seems to be working on upstreaming their changes, but looks like still ongoing
https://github.com/FrameworkComputer/framework-system/blob/main/framework_lib/Cargo.toml#L61